### PR TITLE
Inserting a real data to the Database

### LINKED ISSUE #78: Insert a real data to the Database
Making a migration file that insert a real data to db ( some Articles, few Users, few Subjects).

### PR DESCRIPTION
**Pull request by Shalevro2**
Adding to migration files some data

Creating articles_conent.py for generating  data to articles

Resolve #78

### CODE REVIEW

**Review by Anas-mhameed — COMMENTED** [review]
I Prefer to insert some longer titles in order to test how it will be presented in the UI.

File: projboard/migrations/data/articles_content.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Generated by chat-gpt
```
Comment (Anas-mhameed) [inline projboard/migrations/data/articles_content.py:1]: remove this comment . (;

File: projboard/migrations/0006_test_subject_data.py [hunk 2]
```diff
@@ -14,7 +14,13 @@ def generate_subject_data(apps, schema_editor):
...
         test_data = [
             'Sport',
-            'Math'
+            'Math',
+            'Software',
+            'Medicine and health',
+            'Business',
+            'Education',
+            'Food',
+            'Hobbies',
```
Comment (Anas-mhameed) [inline projboard/migrations/0006_test_subject_data.py:23]: You can add a default one => "Other"

Comment (Shalevro2) [reply projboard/migrations/0006_test_subject_data.py:23]: There is auto option of "other" by django. it's added by default when we create the form

File: projboard/migrations/data/articles_content.py (hunk 1 above)
Comment (Shalevro2) [reply projboard/migrations/data/articles_content.py:1]: This is just a credit to the creators of the data

**Review by Anas-mhameed — COMMENTED** [review]
I think you should move dir data outside the migration and then you can remove the init file , if this didn't work in all cases you should move this dir outside the migration.

File: finalProject_g4/settings.py [hunk 3]
```diff
@@ -23,7 +23,7 @@
 SECRET_KEY = 'django-insecure-xs8-3b9a%2ysxk0fr!vpfqf^o__kltrgep_d39_)jfyxh#8#pu'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+DEBUG = True
```
Comment (rawadabu) [inline finalProject_g4/settings.py:26]: Interesting topics :)
But I haven't figured out why DEBUG changed to True.

**Comment by Shalevro2** [comment]
> I Prefer to insert some longer titles in order to test how it will be presented in the UI.

Dont at [b39938d](https://github.com/redhat-beyond/Smarticle/pull/80/commits/b39938d5bdaf7e098d7aa76661c58c1e8cb91c0a)

File: projboard/migrations/0006_test_subject_data.py (hunk 2 above)
Comment (Anas-mhameed) [reply projboard/migrations/0006_test_subject_data.py:23]: which file and which line does this ?

Comment (Shalevro2) [reply projboard/migrations/0006_test_subject_data.py:23]: take a look at create-article page...

Comment (Anas-mhameed) [reply projboard/migrations/0006_test_subject_data.py:23]: found it

**Review by Anas-mhameed — APPROVED** [review]
LGTM

**Review by NitzanMordhai — APPROVED** [review]

**Review by finziamit — APPROVED** [review]
LGTM

**Review by yaelnoyman1 — APPROVED** [review]
LGTM

**Review by rawadabu — APPROVED** [review]

**Review by avi-biton — APPROVED** [review]
LGTM

**Review by Yarboa — APPROVED** [review]
Approve

**Pull request merged**